README.md

```diff
@@ -1,2 +1,315 @@
+- [ ] Automatic proxy injection on workloads
+- [ ] Functioning of  `linkerd tap`, `stat`, `routes` and `edges` commands
+- [ ] Verifying the functioning of the `tap` extension API server
+- [ ] Retries and timeouts
+- [ ] Data plane health checks
+- [ ] Ingress configuration
+_...and much more_
+
+If you are interested in helping to extend the test suites, see
+[Adding new tests](https://github.com/linkerd/linkerd2-conformance#2-adding-tests)
+below.
+
+## Table of Contents
+
+- [Repository Structure](https://github.com/linkerd/linkerd2-conformance#repository-structure)
+- [Configuring your tests](https://github.com/linkerd/linkerd2-conformance#configuring-your-tests)
+  - [Configuration Options](https://github.com/linkerd/linkerd2-conformance#configuration-options)
+- [Usage](https://github.com/linkerd/linkerd2-conformance#usage)
+  - [Using the Sonobuoy CLI](https://github.com/linkerd/linkerd2-conformance#using-the-sonobuoy-cli)
+  - Running the tests using Docker
+  - [Running the tests locally](https://github.com/linkerd/linkerd2-conformance#running-the-tests-locally)
+- [Adding new tests](https://github.com/linkerd/linkerd2-conformance#adding-new-tests)
+  - [Bootstrapping](https://github.com/linkerd/linkerd2-conformance#1-bootstrapping)
+  - [Adding tests](https://github.com/linkerd/linkerd2-conformance#2-writing-the-tests)
+  - [Wiring up the newly added tests](https://github.com/linkerd/linkerd2-conformance#3-wiring-up-the-newly-added-test)
+
+## Repository Structure
+
+- [`specs`](https://github.com/linkerd/linkerd2-conformance/tree/master/specs)
+contains the tests for each of the features organized into separate
+packages
+- [`sonobuoy`](https://github.com/linkerd/linkerd2-conformance/tree/master/sonobuoy)
+contains the items required to be able to run the tests as a
+[Sonobuoy](https://github.com/vmware-tanzu/sonobuoy) plugin
+- [`utils`](https://github.com/linkerd/linkerd2-conformance/tree/master/utils)
+contains helper functions that can be used while
+writing conformance tests
+- [`bin`](https://github.com/linkerd/linkerd2-conformance/blob/master/bin)
+contains useful helper scripts to build/push the
+Docker image and running the tests
+- [`testdata`](https://github.com/linkerd/linkerd2-conformance/blob/master/bin)
+contains the necessary files required by the tests,
+organized into subfolders
+
+## Configuring your tests
+
+The conformance tests can be easily configured
+by specifying a `config.yaml` that holds the
+configuration values. This includes things like
+Linkerd version, Linkerd add-ons, test specific
+configuration, binary path, etc.  
+
+Run the command below to pull up a sample
+configuration file, `config.yaml`, and modify
+it according to your requirements. The tests
+shall read this YAML file during runtime and run
+accordingly.
+
+```bash
+CONFIG=https://raw.githubusercontent.com/linkerd/linkerd2-conformance/master/config.yaml
+
+curl -sL $CONFIG > config.yaml
+```
+
+### Configuration options
```

Comment:
I would keep the configuration options later in the file, as users don't have to touch this in the default case.